taxonomies/food/categories.txt

```diff
@@ -119559,6 +119548,12 @@ nl: Aardappelbereidingen
 pt: Preparados de batata
 intake24_category_code:en: WFFL
 
+# potato preparation similar to gnocchi, originating from Poland, and also popular in Lithuania and Belarus.
+< en:Potato preparations
+en: Kopytka
+pl: Kopytka
```

Comment:
Maybe add an `xx` entry too, if this is the name it’s commonly known as (in LT and Belarus too)?

```suggestion
< en:Potato preparations
en: Kopytka
xx: Kopytka
pl: Kopytka
```